src/pages/products.js

```diff
@@ -0,0 +1,31 @@
+import React from "react"
+import { graphql } from "gatsby"
+
+export default function ProductSection({ data }) {
+
+  const productSectionData = data.prismicProductPage.data
+
+  return (
+    <div>
+      <h1>Product Page</h1>
+    </div>
+  )
+}
+
+export const query = graphql`
```

Comment:
Could you please demonstrate this is working fine by utilizing all of the pulled in data in the component?